.github/workflows/build-test.yml

```diff
@@ -85,6 +85,7 @@ jobs:
           ASSETS_MANAGER_ADDRESS: ${{secrets.ASSETS_MANAGER_ADDRESS}}
           DID_SYNC_MODE_FULL: false
           STATUS_LIST_DOMAIN: ${{secrets.STATUS_LIST_DOMAIN}}
+          NATS_ENVIRONMENT_NAME: dev
```

Comment:
Why this change in this PR?

Reply:
Because i saw that it has undefined value in nats subject, so I added it